README.md

```diff
@@ -1,77 +1,119 @@
-
-### Submit a store URL
 
-```
-appknox submit_url "<Your store URL Goes here>"
+## Quickstart
+
+```python
+In [1]: import appknox
+
+# Create an Appknox instance and login
+In [2]: client = appknox.Appknox(username='viren', password='foobar')
+In [3]: client.login()
+
+# List projects for authenticated user
+In [4]: client.get_projects()
+Out[4]:
+[Project(id=3, created_on='2017-06-23 07:19:26.720829+00:00', file_count=3,
+         package_name='org.owasp.goatdroid.fourgoats', platform=0,
+         updated_on='2017-06-23 07:26:55.456744+00:00'),
+ Project(id=4, created_on='2017-06-27 08:27:54.486226+00:00', file_count=1,
+         package_name='com.appknox.mfva', platform=0,
+         updated_on='2017-06-27 08:27:54.637432+00:00')]
+
+# List files within a project
+In [5]: client.get_files(4)
+Out[5]: [File(id=6, name='MFVA', version='1.0', version_code='6')]
+
+# Upload a file and commence static scan
+In [6]: client.upload_file(open('/home/viren/apk/mfva_1.0.apk', 'rb'))
+
+# Note the file just uploaded
+In [7]: client.get_files(4)
+Out[7]:
+[File(id=6, name='MFVA', version='1.0', version_code='6'),
+ File(id=7, name='MFVA', version='1.0', version_code='6')]
+
+# Get analyses for the new file
+In [8]: client.get_analyses(6)[:3]
+Out[8]:
+[Analysis(id=267, risk=2, status=3, cvss_base=6.8,
+          findings=[{'title': None, 'description': 'Unprotected service: com.appknox.mfva.ExportedService'}],
+          updated_on='2017-06-27 08:28:35.166608+00:00', vulnerability_id=1),
+ Analysis(id=235, risk=3, status=3, cvss_base=7.3,
+          findings=[{'title': None, 'description': 'pathPrefix=/'}],
+          updated_on='2017-06-27 08:28:35.240543+00:00', vulnerability_id=2),
+ Analysis(id=236, risk=3, status=3, cvss_base=7.7,
+          findings=[{'title': None, 'description': 'Debug enabled within the app'}],
+          updated_on='2017-06-27 08:28:35.296126+00:00', vulnerability_id=3)]
+
+# Note the vulnerability_id for Analysis(id=235). To get details about the vulnerability
+In [9]: client.get_vulnerability(2)
+Out[9]: Vulnerability(name='Improper Content Provider Permissions',
+                      description='A content provider permission was set to allow access from any other app on the device.
+                                   Content providers may contain sensitive information about an app and therefore should not be shared.',
+                      intro="The `ContentProvider` class provides a mechanism for managing and sharing data with other applications.
+                             When sharing a provider's data with other apps, access control should be carefully implemented to prohibit unauthorized access to sensitive data.")
 ```
 
-### Uploading your App
+## Command-line interface
 
 ```
-appknox upload /path/to/your/android.apk
-appknox upload /path/to/your/ios.ipa
+$ appknox login -h https://api.appknox.com
```

Comment:
Then can we remove that from docs? I just want people to get started with as much less lines as possible 😛